### PR TITLE
Munge headerlines

### PR DESCRIPTION
**Pull request by thockin**

### CODE REVIEW

**Comment by thockin** [comment]
This fights with gendocs.  Need to think about it (and this might happen on any such markdown-strictness fix)

**Comment by thockin** [comment]
fixed now

**Comment by thockin** [comment]
rebased

**Comment by thockin** [comment]
@k8s-bot test this please

**Comment by thockin** [comment]
all greennnnnn

File: docs/devel/development.md [hunk 1]
```diff
@@ -133,17 +142,25 @@ export KPATH=$HOME/code/kubernetes
...
 # Or copy your existing local repo here. IMPORTANT: making a symlink doesn't work.
+
 ```
 
 2) Set up your GOPATH.
 
 ```
+
 # Option A: this will let your builds see packages that exist elsewhere on your system.
+
```
Comment (bgrant0607) [inline docs/devel/development.md]: Shouldn't be applied to code blocks

File: docs/roadmap.md [hunk 2]
```diff
@@ -51,9 +52,12 @@ clustered database or key-value store. We will target such workloads for our
 1.0 release.
 
 ## v1 APIs 
+
 For existing and future workloads, we want to provide a consistent, stable set of APIs, over which developers can build and extend Kubernetes. This includes input validation, a consistent API structure, clean semantics, and improved diagnosability of the system. 
 ||||||| merged common ancestors
```
Comment (bgrant0607) [inline docs/roadmap.md]: Is this from an old merge conflict?

**Comment by thockin** [comment]
This catches code blocks.  Dammit.  Back to work

File: docs/roadmap.md (hunk 2 above)
Comment (thockin) [inline docs/roadmap.md]: This is from a merge conflict WEEKS ago.  Sigh.  Anyone have bandwidth to figure out our intentions here :)

**Comment by thockin** [comment]
Now bypasses code blocks

**Comment by thockin** [comment]
@nikhiljindal you might want to follow this pattern or risk the same code-blocks problem

**Comment by bgrant0607** [comment]
LGTM

**Comment by bgrant0607** [comment]
Passed verify-gendocs.sh
cc @davidopp please merge

**Comment by lavalamp** [comment]
LGTM

**Pull request merged**